tests/unit/http/test_mixins.py

```
@@ -1017,11 +1003,12 @@ class Service(AsyncModifiableResourceMixin[DummyModel]):  # noqa: WPS431
     ],
 )
 def test_managed_resource_mixin(method_name):
+    """Test that ManagedResourceMixin has the required methods."""
+
     class ManagedService(ManagedResourceMixin[DummyModel]):  # noqa: WPS431
```

Ruff (0.14.6)

Unused `noqa` directive (unknown: `WPS431`)

Remove unused `noqa` directive

(RUF100)